build.cmd

```diff
@@ -26,6 +26,7 @@ set __SkipTestBuild=
 if "%1" == "" goto ArgsDone
 if /i "%1" == "/?" goto Usage
 if /i "%1" == "x64"    (set __BuildArch=x64&&shift&goto Arg_Loop)
+if /i "%1" == "x86"    (set __BuildArch=x86&&shift&goto Arg_Loop)
```

Comment:
I'm looking at Linux too but this PR is strictly for Windows. Maybe next weekend if nobody else does it before me :smile:.